kolibri/core/assets/src/state/actions.js

```diff
@@ -685,6 +684,17 @@ function updateMasteryAttemptState(store, {
     });
 }
 
+function fetchPoints(store) {
+  if (store.state.core.session.user_id &&
+    store.state.core.session.kind[0] !== UserKinds.SUPERUSER) {
```

Comment:
Yes, I went looking for the getters, as I remembered seeing them... then realized they were on the details-reports branch. Will rebase and use those.